sonar-db/src/main/resources/org/sonar/db/component/SnapshotMapper.xml

```diff
@@ -118,6 +118,14 @@
     ORDER BY e.event_date DESC
   </select>
 
+  <select id="selectSnapshotsOrderedByAscCreatedAt" parameterType="map" resultType="Snapshot">
+    SELECT
+    <include refid="snapshotColumns"/>
+    FROM snapshots s
+    INNER JOIN projects p ON p.id=s.project_id AND p.id=#{componentId}
```

Comment:
Hum... No reason... A copy/paste from selectPreviousVersionSnapshots, which should then also be updated.